src/gui/PhysicalGameStatePanel.java

```diff
@@ -55,7 +55,20 @@
     int last_start_y = 0;
     int last_grid = 0;
 
+    // the color scheme
     int colorScheme = COLORSCHEME_BLACK;
+    static Color GRIDLINE;
+    static Color PLAYER0UNIT_OUTLINE;
+    static Color PLAYER0_PARTIAL_VISIBILITY;
```

Comment:
change "PARTIAL_VISIBILITY" to "PARTIAL_OBSERVABILITY", which is the word used in the rest of the case base :)